llvm/lib/CodeGen/AsmPrinter/DwarfFile.cpp

```diff
@@ -24,6 +24,10 @@ void DwarfFile::addUnit(std::unique_ptr<DwarfCompileUnit> U) {
   CUs.push_back(std::move(U));
 }
 
+void DwarfFile::addTypeUnitSymbol(DwarfTypeUnit &U) {
+  TUSymbols.emplace_back(U.getLabelBegin(), U.getUniqueID());
```

Comment:
Could we store these TUSymbols in the accelerator table instead of adding more wider-scoped state to DwarfDebug?